### ISSUE TITLE
CLI: new output is not buildable

### ISSUE THREAD

**Issue by gedw99** [livefir/fir#41]
using https://github.com/livefir/fir/releases/tag/v0.3.5

```sh
cli new test01

cd test01
go mod init main
go mod tidy
go build .
/Users/apple/go/pkg/mod/github.com/livefir/fir@v0.3.5/markdown.go:332:44: undefined: extension.CJK
```

**Comment by gedw99** [comment]
all the examples get the same markdown extension error on compile btw.

**Comment by adnaan** [comment]
Thank you for bringing this up. It seems `github.com/yuin/goldmark` was old. Updated it and released a new version v0.3.6 which should work now.
